Content.Shared/CCVar/CCVars.cs

```diff
@@ -29,6 +29,12 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<string> RulesHeader =
             CVarDef.Create("server.rules_header", "ui-rules-header", CVar.REPLICATED | CVar.SERVER);
 
+        /// <summary>
+        ///     The token used to authenticate with the admin API. Leave empty to disable the admin API. This is a secret! Do not share!
+        /// </summary>
+        public static readonly CVarDef<string> AdminApiToken =
```

Comment:
1. This CVar is named incorrectly, it should always have the full name in the C# name.
2. Surely you'd want to put this under the `admin` category instead.